analytics/reports/inventory.md

```diff
@@ -0,0 +1,362 @@
+  AND `Received_Date` < STR_TO_DATE('2024-05-08 00:00:00.000000', '%Y-%m-%d %H:%i:%s.%f')
+GROUP BY `SKU`
+ORDER BY `SUM(ABS(Difference))` DESC
+LIMIT 10000;
+```
+
+</details>
+
+### Query Logic
+
+**Data Selection:** Data selection involves gathering essential details from various tables. It selects shipment IDs, product SKUs, quantities, and facility information, providing insights into discrepancies in received shipments. This curated data forms the foundation for the report, enabling effective analysis and resolution of discrepancies.
+
+Joining Relevant Tables:** To compile a comprehensive dataset, the SQL query fetches information about shipments, products, facilities, and statuses. By joining these tables, the query gathers all the necessary information needed to understand discrepancies in received shipments.
+
+**Selecting Necessary Information:** Once the tables are connected, the query selects specific details crucial for the report. It picks essential information like shipment IDs, product SKUs, facility IDs, expected and received quantities, and dates. These details provide insights into where the discrepancies occur, what products are involved, and when the shipments were received. This approach allows the report to focus on the key aspects of receiving discrepancies, enabling users to pinpoint issues and take corrective actions effectively.
+
+**Calculating Differences and Defining Statuses:** Within the subquery, the query calculates the difference between expected and received quantities by subtracting the Quantity Accepted from the quantity. A status is assigned to each shipment item based on this difference, categorizing shipments into `Completed`, `NotReceived`, `OverReceived`, `UnderReceived`, and `Manually added item to transfer order`.
+
+**Grouping Data:** The query groups the data by `Destination_Facility_Id` and `SKU`. This grouping helps to summarize and organize the discrepancies for each product at each facility.
+
+**Ordering:** The query orders the results in descending order based on the total discrepancies, ensuring the product with more discrepancies appears first.
+
+
+## Recorded Variances Report
+
+This report provides retailers with detailed insights into discrepancies in inventory levels across different products and facilities. By highlighting specific product SKUs that have experienced variances and outlining the reasons behind these discrepancies, the report allows retailers to pinpoint areas of concern swiftly and accurately. Understanding the root causes of inventory discrepancies enables retailers to take proactive measures to rectify issues, whether they stem from inaccuracies in recording, theft, damage, or other factors.
+
+This report helps in optimizing inventory management by ensuring that stock levels align with demand and sales forecasts. By identifying variances promptly, retailers can prevent overstocking or stockouts, thus minimizing potential revenue losses.
+
+### Glossary
```

Comment:
Are we naming this headings as Glossary or Report structure? i believe glossary is the better term